packages/app/obojobo-repository/server/events.js

```diff
@@ -33,6 +33,7 @@ oboEvents.on(DraftModel.EVENT_DRAFT_DELETED, () => {
 			AND
 			drafts.deleted = TRUE`
 		)
+		.then(logger.log(`User ${draft.userId} deleted module ${draft.id}`))
```

Comment:
I told you the wrong function to use - it's "info", not "log"

```suggestion
		.then(logger.info(`User ${draft.userId} deleted module ${draft.id}`))
```